classes/class-plugin.php

```diff
@@ -112,6 +115,27 @@ public function __construct() {
 		$this->wc_admin_customers = new Woocommerce\AdminCustomers();
 		$this->wc_my_account = new Woocommerce\MyAccount();
 		$this->oembed = new OEmbed\OEmbed();
+		// Collect all modules and filter them based on the whitelist if available.
+		$this->modules = [
+			'email' => $this->email,
+			'hovercards' => $this->hovercards,
+			'avatar' => $this->avatar,
+			'proxy' => $this->proxy,
+			'quick_editor' => $this->quick_editor,
+			'analytics' => $this->analytics,
+			'block' => $this->block,
+			'patterns' => $this->patterns,
+			'wc_admin_customers' => $this->wc_admin_customers,
+			'wc_my_account' => $this->wc_my_account,
+			'oembed' => $this->oembed,
+		];
+		$modules_whitelist = apply_filters( 'gravatar_enhanced_modules_whitelist', null);
+		if ( is_array($modules_whitelist) ) {
+			$this->modules = array_intersect_key($this->modules, array_flip($modules_whitelist));
+		}
+
+		// Handles the discussions settings page
+		$this->discussions = new Options\DiscussionsPage( $this->auto_options, $this->lazy_options, $enabled_modules = array_keys($this->modules) );
```

Comment:
We're currently supporting PHP 7.4 and I think named parameters are PHP 8+

Reply:
I don't know why this was added. It's not even named parameters, it's an unused variable declaration.

Removed!